Src/Service/Service/persistence/hbm/volunteerOpportunity.hbm.xml

```diff
@@ -5,10 +5,17 @@
             <generator class="identity" />
         </id>
 
-        <many-to-one name="CreatedBy" column="CreatedByUserRecordId" class="User" cascade="all-delete-orphan" />
+        <many-to-one name="CreatedBy" column="UserRecordId" class="User"/>
+        <many-to-one name="Organization" column="OrganizationRecordId" class="Organization" cascade="all-delete-orphan"/>
+      
+        <property name="ProjectName" length="100"/>

```

Comment:
These might be able to be removed and just reference the ProjectRecordId as an associate relationship. Depending on what you decide with earlier comments.